datajunction-server/tests/examples.py

```diff
@@ -718,7 +718,7 @@
         ("/nodes/default.repair_orders_fact/link"),
         {
             "dimension_node": "default.municipality_dim",
-            "join_type": "left",
+            "join_type": "inner",
```

Comment:
I can't make sense why all these `left` joins are converting to `inner` across the unit tests... any ideas?

Reply:
These are all inner joins now because of the expectations of the new build -- remember the discussion on how we always need to filter inside the dimension node first? And that left vs inner joins should actually have different effects on how the dimension is brought in -- if it's an inner join, it implies that the transform node and the dimension node have the same cardinality of dimensions, so it's fine to basically introduce a "where clause" to only include that dimension in the outer-level of the query. Otherwise, with a left join, only the dimension or fact will get filtered to the right values, which means that the final joined output can still have additional unexpected values that weren't filtered out.